spec/fabricators/keyword_fabricator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Fabricator(:keyword) do
+  keyword 'MyString'
+  status ''
+  user nil
```

Comment:
Should we create user fabricator and assign the user here?

Reply:
Fixed in [`f42675d` (#40)](https://github.com/suho/google-search-ruby/pull/40/commits/f42675dd1e9b7fdfe0b06e96e5e8709d750dd695)